pkg/network/usm/sharedlibraries/watcher.go

```diff
@@ -42,6 +41,10 @@ func ToBytes(l *LibPath) []byte {
 	return l.Buf[:l.Len]
 }
 
+func ToString(l *LibPath) string {
+	return string(ToBytes(l))
+}
```

Comment:
I do recommend to modify the signature to 
```go
func (l *LibPath) String() string
```
It will mean the type implemented the Stringer interface, and will allow you to use `%s` format